tests/test_ner4opt.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+#  -*- coding: utf-8 -*-
+import json
+import math
+import os
+import unittest
+from pathlib import Path
+
+from ner4opt import Ner4Opt
+from ner4opt.exceptions import InvalidModelError
+
+
+class Ner4OptTest(unittest.TestCase):
```

Comment:
you have a test, test_quick_start() or sth like that, to test the quick start example from the readme, right?

Reply:
Yes, example in the readme is covered in the test cases.